project1/index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>The Crane Wife: Short Story by CJ Hauser_Project 1</title>
+    <link rel="stylesheet" href="styles.css">
+    <link
+        href="https://fonts.googleapis.com/css?family=Fjalla+One|Montserrat:300,300i,400,600,600i|Satisfy&display=swap&subset=latin-ext"
+        rel="stylesheet">
+</head>
+
+
+<body>
+
+    <main>
+
+
+        <!-- I'm not sure if I am saving my commits and the file properly-->
+        <header style="text-align: center">
+            <h1>The Crane Wife</h1>
+            <p class="anchor"><a href="https://cjhauser.com/about-us">By CJ Hauser</a></p>
+            <p class="date">July 16, 2019</p>
+        </header>
+
+
+        <blockquote>
+            <p>When I looked at that mouse with her broom, I wondered which one of us was wrong
+                about who
+                <strong><em>I</em></strong> was.</p>
+        </blockquote>
+
+        <section>
+            <p>Ten days after I called off my engagement I was supposed to go on a scientific expedition to study the
+                whooping crane on the gulf coast of Texas. Surely, I will cancel this trip, I thought, as I shopped for
+                nylon hiking
+                pants that zipped off at the knee. Surely, a person who calls off a wedding is meant to be sitting sadly
+                at
+                home, reflecting on the enormity of what has transpired and not doing whatever it is I am about to be
+                doing
+                that
+                requires a pair of plastic clogs with drainage holes. Surely, I thought, as I tried on a very large and
+                floppy hat
+                featuring a pull cord that fastened beneath my chin, it would be wrong to even be wearing a hat that
+                looks
+                like
+                this when something in my life has gone so terribly wrong.</p>
+
+            <p>Ten days earlier I had cried and I had yelled and I had packed up my dog and driven away from the upstate
+                New
+                York house with two willow trees I had bought with my fiancé.</p>
+
+            <p>Ten days later and I didn’t want to do anything I was supposed to do.</p>
+        </section>
+
+        <div>
+            . . .
```

Comment:
Hi, @ChantelH!

Some of your layout problems are because these are periods (`.`) -- and, by their very nature, periods are aligned to the bottom of a sentence. Since what you want here is dots that are more in the middle of a sentence, I'd recommend using bullets (like in bulleted/unordered lists). You can do this by replacing the periods with the named special symbol for bullet, like so:

```suggestion
            &bull; &bull; &bull;
```

Each `&bull;` creates a dot, essentially, but this time one that is vertically aligned to the center of the text. This should give you this result (top is bullets; bottom is the periods you are currently using):

![image](https://user-images.githubusercontent.com/2653980/65542031-6e86a500-dedc-11e9-8523-2d4b95ecf7fe.png)

You'll want to adjust the `font-size` to be more subtle since bullets are larger typographic entities than the periods. 😀